_install-and-configure/install-opensearch/windows.md

```diff
@@ -65,6 +65,12 @@ An OpenSearch node in its default configuration (with demo certificates and user
          ```
          {% include copy.html %}
 
+      1. Set the initial admin password via the `OPENSEARCH_INITIAL_ADMIN_PASSWORD` environment variable.
+         ```bat
+         set OPENSEARCH_INITIAL_ADMIN_PASSWORD=< Admin password >
+         ```
+         {% include copy.html %}
+
       1. Run the batch script.
```

Comment:
this step should be numbered as 2

Reply:
All of them are numbered as 1

Reply:
Markdown automatically will number these in sequence. So if you put "1. 1. 1.", the output will be "1. 2. 3"